### PR TITLE
ACoRD: - fix link for software steps (see desc):

### PR DESCRIPTION
**Pull request by IrisDroidology**
Lines 6 + 7:
### Installing
To install and run the updated software in this branch, please follow the instructions in the [Software Steps.pdf document in the ros-rework branch of the master repository](https://github.com/nasa-jpl/open-source-rover/blob/differential-pivot-rework/Software/Software%20Steps.pdf.

Fix link to the pdf

### CODE REVIEW

**Comment by IrisDroidology** [comment]
The link sent you to a branch that had either not been created or had been deleted.

Liam

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This branch is an in-development version of the Open Source Rover Code. It conta
 ## ROS
 
 ### Installing
-To install and run the updated software in this branch, please follow the instructions in the [Software Steps.pdf document in the ros-rework branch of the master repository](https://github.com/nasa-jpl/open-source-rover/blob/ros-rework/Software/Software%20Steps.pdf).
+To install and run the updated software in this branch, please follow the instructions in the [Software Steps.pdf document in the ros-rework branch of the master repository](https://github.com/nasa-jpl/open-source-rover/blob/differential-pivot-rework/Software/Software%20Steps.pdf.
```
Comment (IrisDroidology) [inline README.md]: The link was updated here

Also, I'm adding more comments and updating the code to make it more user-friendly in my next PR/MR in this repo

**Comment by mikcox** [comment]
We've actually merged in this software document into the master branch on the open-source-rover repo.  I've merged PR #29 which addresses this, so this issue is now resolved.  Thanks for submitting the PR!

**Comment by IrisDroidology** [comment]
Thanks @mikcox, glad to see that #29 was resolved. Happy to help!

P.S. The OSR challenge was excellent, excited to implement my rl-policy when I continue working on my OSR!

**Pull request closed without merging**